src/components/BookSummary/BookSummary.tsx

```diff
@@ -6,7 +6,7 @@ import {
   FONT_VARIANTS,
   PURPOSE,
 } from '@/components/Button/Button.types';
-import { SlidePicture } from '@/components/SlidePicture/SlidePicture';
+import { StoryPagePicture } from '@/components/StoryPagePicture/StoryPagePicture';
```

Comment:
add to index.ts in `StoryPagePicture` folder and then
`import StoryPagePicture from '@/components/StoryPagePicture';`

Reply:
done